src/openfl/display/BitmapData.hx

```diff
@@ -171,6 +174,18 @@ class BitmapData implements IBitmapDrawable {
 		
 		image.colorTransform (rect.__toLimeRectangle (), colorTransform.__toLimeColorMatrix ());
 		
+		__markUsersRenderDirty ();
```

Comment:
this is the call we need to add to all methods that mutate the image

Reply:
Doesn't this need to stay in sync with `imageVersion++`

Reply:
Yes. The problem is that there's currently no single place where `image.version++` is done, so we'll have to rework this too...